phpBB/phpbb/notification/method/email.php

```diff
@@ -68,6 +98,53 @@ public function is_available(type_interface $notification_type = null)
 		return parent::is_available($notification_type) && $this->config['email_enable'] && $this->user->data['user_email'];
 	}
 
+	/**
+	* {@inheritdoc}
+	*/
+	public function get_notified_users($notification_type_id, array $options, $notification_type_name)
+	{
+		$notified_users = array();
+
+		if ($notification_type_name == 'notification.type.post' && !empty($options['item_parent_id']))
+		{
+			// Topics watch
+			$sql = 'SELECT tw.user_id
+					FROM ' . $this->topics_watch_table . ' tw
+					LEFT JOIN ' . $this->topics_track_table . ' tt
+						ON (tt.user_id = tw.user_id AND tt.topic_id = tw.topic_id)
+					LEFT JOIN ' . $this->posts_table . ' p
+						ON (p.topic_id = tw.topic_id)
+					WHERE tw.topic_id = ' . (int) $options['item_parent_id'] . '
+						AND p.post_time > tt.mark_time
+					HAVING COUNT(p.post_id) > 1';
```

Comment:
HAVING without GROUP BY is not allowed in all MySQL configuration

Reply:
Not allowed in SQL Server at all.

Reply:
😭